backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -444,11 +445,9 @@ def upload_for_ide(self, request: HttpRequest, pk: Any = None) -> Response:
         return Response({"data": documents})
 
     @action(detail=True, methods=["delete"])
-    def delete_for_ide(self, request: HttpRequest, pk: Any = None) -> Response:
+    def delete_for_ide(self, request: HttpRequest, pk: uuid) -> Response:
         custom_tool = self.get_object()
-        serializer = FileInfoIdeSerializer(data=request.GET)
-        serializer.is_valid(raise_exception=True)
```

Comment:
@johnyrahul Why is the serializer validation removed?

Reply:
@Deepak-Kesavan  the above serializer is expecting type as well. For this particular API it's only expecting document ID inside the body. I will go ahead and create a new serializer

Reply:
@Deepak-Kesavan  addressed